phpunit/experimental/interactivity-api/directive-processing-test.php

```diff
@@ -10,9 +10,6 @@
 
 
 class Helper_Class {
```

Comment:
I think that we should get rid of this class and just add the helpers to the main class.

_Once you make this change, please remove the `@phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound` comment at the top._

Reply:
Done!